WordPress/Classes/WordPress.xcdatamodeld/.xccurrentversion

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>_XCCurrentVersionName</key>
-	<string>WordPress 74.xcdatamodel</string>
+	<string>WordPress 75.xcdatamodel</string>
```

Comment:
When you update the core data model, make sure you add an entry to `MIGRATIONS.md` to note what changed in the model 👍

Reply:
Ok thanks for the hint!